Description: Used canned example for AlgebraicSplines documentation
 Otherwise, we get a SIGSEGV when building on i386.
Bug: https://github.com/Macaulay2/M2/issues/1429
Author: Doug Torrance <dtorrance@piedmont.edu>
Last-Update: 2020-10-27

--- a/M2/Macaulay2/packages/AlgebraicSplines.m2
+++ b/M2/Macaulay2/packages/AlgebraicSplines.m2
@@ -2505,11 +2505,40 @@
 	Text
 	    The above example shows that the Billera-Schenck-Stillman spline complex is the cokernel of the natural map between the complex
 	    of ideals (given by idealsComplex) and the cellular chain complex of $\Delta$ relative to its boundary.
-	Example
-	    V = {{1, 0, 0}, {-1, 0, 0}, {0, 1, 0}, {0, -1, 0}, {0, 0, 1}, {0, 0, -1}, {-2, -2, -2}, {-2, 2, -2}, {2, 2, -2}, {2, -2, -2}, {-2, -2, 2}, {-2, 2, 2}, {2, 2, 2}, {2, -2, 2}};
-	    F = {{0, 1, 2, 3, 4, 5}, {0, 8, 9, 12, 13}, {1, 6, 7, 10, 11}, {2, 7, 8, 11, 12}, {3, 6, 9, 10, 13}, {4, 10, 11, 12, 13}, {5, 6, 7, 8, 9}, {0, 2, 8, 12}, {0, 3, 9, 13}, {0, 4, 12, 13}, {0, 5, 8, 9}, {1, 2, 7, 11}, {1, 3, 6, 10}, {1, 4, 10, 11}, {1, 5, 6, 7}, {2, 4, 11, 12}, {3, 4, 10, 13}, {3, 5, 6, 9}, {2, 5, 7, 8}, {0, 2, 4, 12}, {0, 2, 5, 8}, {0, 3, 4, 13}, {0, 3, 5, 9}, {1, 2, 4, 11}, {1, 2, 5, 7}, {1, 3, 4, 10}, {1, 3, 5, 6}};
-	    C = splineComplex(V,F,1);
-	    associatedPrimes annihilator HH_2 C
+	CannedExample
+	    i25 : V = {{1, 0, 0}, {-1, 0, 0}, {0, 1, 0}, {0, -1, 0}, {0, 0, 1}, {0, 0, -1}, {-2, -2, -2}, {-2, 2, -2}, {2, 2, -2}, {2, -2, -2}, {-2, -2, 2}, {-2, 2, 2}, {2, 2, 2}, {2, -2, 2}};
+
+	    i26 : F = {{0, 1, 2, 3, 4, 5}, {0, 8, 9, 12, 13}, {1, 6, 7, 10, 11}, {2, 7, 8, 11, 12}, {3, 6, 9, 10, 13}, {4, 10, 11, 12, 13}, {5, 6, 7, 8, 9}, {0, 2, 8, 12}, {0, 3, 9, 13}, {0, 4, 12, 13}, {0, 5, 8, 9}, {1, 2, 7, 11}, {1, 3, 6, 10}, {1, 4, 10, 11}, {1, 5, 6, 7}, {2, 4, 11, 12}, {3, 4, 10, 13}, {3, 5, 6, 9}, {2, 5, 7, 8}, {0, 2, 4, 12}, {0, 2, 5, 8}, {0, 3, 4, 13}, {0, 3, 5, 9}, {1, 2, 4, 11}, {1, 2, 5, 7}, {1, 3, 4, 10}, {1, 3, 5, 6}};
+
+	    i27 : C = splineComplex(V,F,1);
+
+	    i28 : associatedPrimes annihilator HH_2 C
+
+	    o28 = {ideal (t , t , t ), ideal (t , t , t ), ideal (t , t , t ), ideal (t ,
+	                   3   2   0           3   2   1           3   1   0           2
+	          -----------------------------------------------------------------------
+	          t , t  - t ), ideal (t , t , t  + t ), ideal (t , t  - t , t ), ideal
+	           1   0    3           2   1   0    3           2   1    3   0
+	          -----------------------------------------------------------------------
+	          (t , t  + t , t ), ideal (t  - t , t , t ), ideal (t  + t , t , t ),
+	            2   1    3   0           2    3   1   0           2    3   1   0
+	          -----------------------------------------------------------------------
+	          ideal (t  - 2t , t  - 2t , t  - 2t ), ideal (t  - 2t , t  - 2t , t  +
+	                  2     3   1     3   0     3           2     3   1     3   0
+	          -----------------------------------------------------------------------
+	          2t ), ideal (t  - 2t , t  + 2t , t  - 2t ), ideal (t  - 2t , t  + 2t ,
+	            3           2     3   1     3   0     3           2     3   1     3
+	          -----------------------------------------------------------------------
+	          t  + 2t ), ideal (t  + 2t , t  + 2t , t  - 2t ), ideal (t  + 2t , t  +
+	           0     3           2     3   1     3   0     3           2     3   1
+	          -----------------------------------------------------------------------
+	          2t , t  + 2t ), ideal (t  + 2t , t  - 2t , t  - 2t ), ideal (t  + 2t ,
+	            3   0     3           2     3   1     3   0     3           2     3
+	          -----------------------------------------------------------------------
+	          t  - 2t , t  + 2t ), ideal (t , t , t , t )}
+	           1     3   0     3           3   2   1   0
+
+	    o28 : List
 	Text
 	    The above example showcases a fairly complex three dimensional polyhedral complex.
 	    It is a three-dimensional analog of the Morgan-Scot partition in the sense that
